profiles/tests/test_audit_log.py

```diff
@@ -42,17 +41,19 @@ def assert_common_fields(
     actor_role="SYSTEM",
     target_profile_part="base profile",
 ):
-    now = get_unix_timestamp_now()
+    now_dt = datetime.now(tz=timezone.utc)
+    now_ms_timestamp = int(now_dt.timestamp() * 1000)
+
     audit_event = log_message["audit_event"]
 
     assert audit_event["origin"] == "PROFILE-BE"
     assert audit_event["status"] == "SUCCESS"
     assert audit_event["operation"] == operation
     assert audit_event["actor"]["role"] == actor_role
 
-    assert_almost_equal(audit_event["date_time_epoch"], now)
+    # 50ms limit was arrived to by checking the maximum time at local laptop
+    assert_almost_equal(audit_event["date_time_epoch"], now_ms_timestamp, 50)
```

Comment:
The comment is quite obvious. These kind of leeways usually come from the hat more or less. No need to state it out loud.

But you could put the magic number 50 into a well named constant. And perhaps also use the same on the line 60 in the other timestamp check.